### ISSUE TITLE
Not synchronized between copyright file and some text files

### ISSUE THREAD

**Issue by oo13** [endless-sky/endless-sky#5346]
**Is your feature request related to a problem?
Some data files shows someone else is the copyright holder but copyright file shows only MZ.

- hai/hai news.txt
- human/marauders.txt
- human/frontier jobs.txt
- ka'het/ka'het.txt
- ka'het/ka'het ships.txt
- ka'het/ka'het outfits.txt
- ka'het/ka'het missions.txt
- korath/korath news.txt
- remnant/remnant missions.txt
- remnant/remnant news.txt
- sheragi/sheragi ships.txt
- sheragi/archaeology missions.txt
- sheragi/sheragi outfits.txt

**Related Issue Links**
I can't find it.

**Describe the solution you'd like**
Modify the catch-all element to this:

Files: *
Copyright: MZ **, devs, and contributers.**
License: GPL-3+

**Describe alternatives you've considered**
Append those copyright holders to copyright file.

**Additional context**
Generally a text file has many copyright holders. IMO, the first author is not very important.

**Comment by tehhowch** [comment]
If both solutions are compatible with the machine readable format, then either is fine for me.

**Comment by Zitchas** [comment]
In terms of the first solution, would that not turn into basically a list of almost everyone who has contributed to ES? modifications to text files are the simplest contributions, and I suspect that most of the 150+ contributors have changed at least one. Example:
```
Files: data/*
Copyright: 
 Michael Zahniser <mzahniser@gmail.com>
 Amazinite ...
 Tehhowch ...
 Pointedstick ...
 contributor 1 ...
 contributer 2 ...
 etc.
License: CC-BY-SA-4.0
```
Having a copyright entry that states all that has two problems. The first being that it effectively assigns every contributor a more or less equal copyright claim on every text file. Most of us (Devs excluded) have not even touched every data file, let alone made noteworthy additions thereto. 

The second being it will make for a very long entry, one that will probably require us to contact every contributor to find out a) what their preferred name for the entry is, and b) what contact info (if any) they want attached to it. This isn't as important as the attribution element, but it could still be quite significant.


The alternative, listing files attached to those who have worked on them is probably a clearer attribution, but is also complex. Do we list everyone who has changed so much as a word in a text file? Is there a percentage threshold that must be met to have a name added to the copyright? The former seems both lax and confusing: I know I've made single word corrections to quite a few files, and I don't think that a handful of corrections warrants my name on the copyright for all those files. On the other hand, percentages seem hard to measure, especially with something that grows over time. Do we leave it as just falling under MZ's heading until we reach the point where we agree that the contributor has contributed a significant portion of the content of that file? And is that significant portion a percentage of the total file, a certain number of lines, or a certain number of objects (ex. Missions, ships, outfits, planets, etc)?

For listing specific files that warrant having a specific author attached to it, would it be something like this?

```
Files:
 data/remnant/remnant*
 data/remnant/remnant?missions*
 data/remnant/remnant?news*
 data/remnant/remnant?outfits*
 data/remnant/remnant?ships*
Copyright: Zitchas (zitchas.jma@gmail.com)
License: CC-BY-SA-4.0
Comment: Derived from works by Michael Zahniser (under the same license).
```

(Technically, I created the entire news file, but it is still based on the format that Michael established in other places, so it might count as derived)

Or would we list each file individually with a list of people who have contributed to it?
```
Files: data/remnant/remnant?missions*
Copyright: 
  Michael Zahniser <mzahniser@gmail.com>
  Zitchas (zitchas.jma@gmail.com)
  Amazinite (...)
License: CC-BY-SA-4.0
```

**Comment by oo13** [comment]
I suggest to add "devs and contributers," not to list all peoples. (I hope you rewrite more appropriate words because I have poor English.)

For example, Hedgewars contains this copyright file:

```
Files: *
Copyright:
        2004-2018 Andrey Korotaev <unC0Rr@gmail.com>
        2006-2008 Igor Ulyanov aka Displacer <iulyanov@gmail.com>
        2004-2015 Hedgewars Project
License: GPL-2

...snip...
```

I use "devs and contributers" instead of "ABC Project" bacause we have no such organization.

**Comment by oo13** [comment]
Solved by #5380.
